### PR TITLE
Add Reminder

### LINKED ISSUE #447: UI for reminders (AdminDashboard)
`P3 - ajout de deux notifs reminders`

### PR DESCRIPTION
**Pull request by MattiaPrimavera**
- Single add reminder selector
- hide selector when editing room (allow setting reminders only at creation)
fix #447
- focus on French language only

### CODE REVIEW

**Review by HugoGresse — APPROVED** [review]

File: src/pages/AdminDashboard/Reminders/NotificationSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            margin: theme.spacing(1),
+            minWidth: 120,
+        },
+        selectEmpty: {
+            marginTop: theme.spacing(2),
+        },
+    })
+)
+
+type Unit = 'Minutes' | 'Heures' | 'Jours'
```
Comment (ntchou) [inline src/pages/AdminDashboard/Reminders/NotificationSelector.tsx:25]: duplicate types with https://github.com/Instant-Visio/InstantVisio-WebApp/pull/469/files#diff-15807506e79bba71e0459579a21b8d6e47edcd6fd9858de4353096510b11b3efR43

File: src/pages/AdminDashboard/CreateRoomForm.tsx [hunk 2]
```diff
@@ -39,6 +40,12 @@ export interface Room {
     roomUrl: string
 }
 
+type Unit = 'Minutes' | 'Heures' | 'Jours'
```
Comment (ntchou) [inline src/pages/AdminDashboard/CreateRoomForm.tsx:43]: maybe in english. You don't use it for displaying

File: src/pages/AdminDashboard/CreateRoomForm.tsx [hunk 3]
```diff
@@ -39,6 +40,12 @@ export interface Room {
...
+interface Notification {
```
Comment (ntchou) [inline src/pages/AdminDashboard/CreateRoomForm.tsx:44]: Move it to notification it's related to `Notification` domain

File: src/pages/AdminDashboard/CreateRoomForm.tsx [hunk 4]
```diff
@@ -224,7 +252,9 @@ const CreateRoomForm = ({ fields, onFormSubmit, onCreateFormReset }) => {
                     id: fields.id,
                     destinations,
                 }
-                onFormSubmit(room, isEditing)
+
+                const remindAt = getRemindAt(room.startAt / 1000, notification)
```
Comment (ntchou) [inline src/pages/AdminDashboard/CreateRoomForm.tsx:256]: suggestion: for better readability use a var `const startAtInMillisecond = room.startAt / 1000`

File: src/pages/AdminDashboard/Reminders/NotificationSelector.tsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        event: React.ChangeEvent<{ value: unknown }>
+    ) => {
+        setNotification({
+            ...notification,
+            number: event.target.value as number,
+        })
+    }
+
+    return (
+        <div>
```
Comment (ntchou) [inline src/pages/AdminDashboard/Reminders/NotificationSelector.tsx:49]: suggestion use react fragment `<> `

**Pull request merged**